pipelines/rj_ssm__celular_seguro/flow.py

```
@@ -0,0 +1,33 @@
+from prefect import flow
+import os
+
+@flow(log_prints=True)
+def rj_ssm__celular_seguro():
+    url = getenv_or_action("API_SINESP__ACESS_TOKEN_URL")
+    # api_key = getenv_or_action("API_SINESP__CLIENT_ID")
+    client_id = getenv_or_action("API_SINESP__CLIENT_ID")
+    client_secret = getenv_or_action("API_SINESP__CLIENT_SECRET")
+
+    print("Testando diretamente pelo CURL")
+    result = os.system(f"curl -v --connect-timeout 5 {url}")
```

Ruff (0.15.20)

[error] Starting a process with a shell, possible injection detected

(S605)